### PR TITLE
Loot Wraith Continued

### PR DESCRIPTION
**Pull request by Setnour6**
I allow everyone that wants to contribute to the new loot wraith changes to commit to the forked repo under my name, on the branch loot-wraith-2-continue.
**the many files changed are a result of me merging the branch to the latest master version**

### What are the specifics of what is being added? 

This PR is a continuation of #384. The differences are that I decided to take the initiative to make the needed changes as part of the review.

### Is there an associated suggestion or dev server #code-needed post? How well does this PR follow it?

see #384. If you want, I can re-target the PR to master.

### Is there anything left to be done in this addition?

I think this is good, like Graydee said.

That damn pineapple... if only he was still around. I miss him.

### CODE REVIEW

**Comment by ScalarVector1** [comment]
Please have this target the master branch directly instead of the old branch, as this manifests as every  single change on master since that branch was created being needed to be reviewed

**Comment by Setnour6** [comment]
> Please have this target the master branch directly instead of the old branch, as this manifests as every single change on master since that branch was created being needed to be reviewed

I changed the targeting!

File: Content/WorldGeneration/WorldgenWorld.cs [hunk 1]
```diff
@@ -19,6 +19,8 @@ public override void ModifyWorldGenTasks(List<GenPass> tasks, ref double totalWe
 			if (ShiniesIndex != -1)
 			{
 				tasks.Insert(DesertIndex + 1, new PassLegacy("Starlight River Temples", UndergroundTempleGen));
+				tasks.Insert(DesertIndex + 4, new PassLegacy("Starlight River Vitric Desert", VitricGen));
```
Comment (ScalarVector1) [inline Content/WorldGeneration/WorldgenWorld.cs:22]: Why are we generating the desert twice?

**Review by ScalarVector1 — CHANGES_REQUESTED** [review]
Looks fine sans the strange introduction of a second generation of the vitric desert. I will also ask that you update PATCH_NOTES.txt to reflect the changes made.

File: Content/WorldGeneration/WorldgenWorld.cs (hunk 1 above)
Comment (Setnour6) [reply Content/WorldGeneration/WorldgenWorld.cs:22]: > Why are we generating the desert twice?

I forgot to remove that line. I'll make that change now.

**Comment by Setnour6** [comment]
> Looks fine sans the strange introduction of a second generation of the vitric desert. I will also ask that you update PATCH_NOTES.txt to reflect the changes made.

Since the previous PR I made involved an audio fix and I didn't update the patch notes (I wasn't sure if the change was significant or not to add), how can I word it properly?
My guess is that it could say "Fixed Combat Shrine Shadows' death sound cutting off"

**Review by ScalarVector1 — APPROVED** [review]

**Comment by ScalarVector1** [comment]
Looks good

**Pull request merged**